test/widgets/WTableTest.C

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright (C) 2010 Emweb bvba, Kessel-Lo, Belgium.
+ *
+ * See the LICENSE file for terms of use.
+ */
+#include <boost/test/unit_test.hpp>
+
+#include <Wt/WApplication.h>
+#include <Wt/WTable.h>
+#include <Wt/Test/WTestEnvironment.h>
+
+using namespace Wt;
+
+namespace {
+Wt::Test::WTestEnvironment environment;
+Wt::WApplication testApp(environment);
+
+void CheckRowNumbers(WTable* table)
```

Comment:
```suggestion
void checkRowNumbers(WTable* table)
```

Wt uses `lowerCamelCase` for functions.